SayTheirNamesTests/ServiceTests/NetworkRequestorTests.swift

```diff
@@ -73,15 +73,15 @@ final class NetworkRequestorTests: XCTestCase {
         wait(for: [requestExpectation], timeout: 2)
     }
 
-    func test_fetchDecodable_propagatesAFError() {
+    func test_fetchDecodable_propagatesSwiftError() {
         guard let url = URL(string: Environment.serverURLString) else { XCTFail("URL was not valid"); return }
         Mock(url: url, dataType: .json, statusCode: 200, data: [.get: Data()]).register()
 
         // Make request, and expect the error to be returned
         let requestExpectation = expectation(description: "Request should return")
-        sut.fetchDecodable(url.absoluteString) { (result: Result<TestObject, AFError>) in
+        sut.fetchDecodable(url.absoluteString) { (result: Result<TestObject, Swift.Error>) in
             switch result {
-            case .failure(.responseSerializationFailed(reason: .inputDataNilOrZeroLength)):
+            case .failure:
```

Comment:
it should be possible to do something like `case .failure(let error as AFError) where error == .responseSerializationFailed(reason: .inputDataNilOrZeroLength)` (I might be off in syntax). It's still valuable to check that error was propagated correctly - that was the purpose of this test function at least :)

Reply:
oh my bad :D

Reply:
> it should be possible to do something like `case .failure(let error as AFError) where error == .responseSerializationFailed(reason: .inputDataNilOrZeroLength)` (I might be off in syntax). It's still valuable to check that error was propagated correctly - that was the purpose of this test function at least :)

unfortunately didn't manage to do it with this syntax

Reply:
no worries 👌 
btw, the home view model file is back 👀

Reply:
but managed to do it by error pattern matching

Reply:
> no worries 👌
> btw, the home view model file is back 👀

I thought you'd need the changes here for the home-pagination branch
I can delete it if that's what should be done 😄

Reply:
I'll change the file there, it's not needed here yet 🙂

Reply:
Great will remove the file from here then 😄